### ISSUE TITLE
Scan POST requests

### ISSUE THREAD

**Issue by c0rv4x** [tristanlatr/burpa#3]
Hey!

Is there an option to scan POST requests? Couldn't find it in the docs

Thanks!

**Comment by tristanlatr** [comment]
Hi @c0rv4x ,

I assume you mean scanning a form URL with the Burp Suite Intruder ? I don't think that's part of the APIs.

Burpa is only a wrapper around existing Burp Suite APIs. 

Currently, burpa uses the official REST API to launch the scan, the API help looks like that:

<img width="1295" alt="Screen Shot 2021-06-15 at 3 13 54 PM" src="https://user-images.githubusercontent.com/19967168/122110240-4fd0ec00-cdec-11eb-91bf-c0c59704432e.png">

So if you think there is something to do, please be more specific regarding what exactly :)

Thanks,

**Comment by Regala** [comment]
From what I remember you can do this by:

- Proxying the POST request into burp
- Issue a scan for that URL; since it's in the proxy history/sitemap it should be picked up and scanned

**Comment by c0rv4x** [comment]
@tristanlatr yeah dude, their api doesn't support that. I was hoping you found a nice workarount
@Regala wow. that would be great. Will try it. Thanks!

**Comment by Regala** [comment]
np lemme know if you confirm that works so I can start using this
project too :P

**Comment by c0rv4x** [comment]
I am not planning to use burpa at the moment as we have our own implementation of this thing, but it seems that your method doesn't really work. I have added POST request and started scanning that URL and the POST request is not active scanned :(
